changelogs/2.11.5.md

```diff
@@ -0,0 +1,149 @@
+  load a row from a run file (gh-10512).
+* Lowered the level used for logging range dump, compaction, split, and
+  coalesce events from `INFO` to `VERBOSE` (gh-10524).
+* Fixed a bug when a compaction task could crash on a disk read error
+  (gh-10555).
+* Fixed a bug when `index.select()` executed in the `read-confirmed`
+  transaction isolation mode (default for read-only transactions) could corrupt
+  the tuple cache by creating a chain bypassing an unconfirmed tuple. The bug
+  could lead to a crash or invalid query results (gh-10558).
+* Added a fiber slice check to `index.count()` to prevent it from blocking
+  for too long while counting tuples in a space stored in memory (gh-10553).
+* Fixed a bug when an attempt to alter the primary index of an empty space
+  triggered a crash if executed concurrently with a DML request (gh-10603).
+* Fixed a bug when `index.stat()` and `index.len()` could report a wrong number
+  of in-memory statements for a non-unique multi-key index of a space with
+  the `defer_deletes` option enabled (gh-10751).
+
+### Replication
+
+* Fixed a memory leak on disconnection from replica (gh-10480).
+* Optimized synchronous transaction queue processing (gh-9917).
+
+
+### LuaJIT
+
+Backported patches from the vanilla LuaJIT trunk (gh-10199). The following
+issues were fixed as part of this activity:
+
+* Now 64-bit non-FAT Mach-O object files are generated via `-b -o osx`.
+* Fixed `string.format()` compilation with many elements.
+* Fixed `dlerror()` in FFI call returning `NULL`.
+* Fixed `__tostring` metamethod access to enum cdata value.
+* Fixed limit check in narrowing optimization.
+* Dropped finalizer table rehashing after GC cycle (gh-10290).
+* Fixed recording of `select(string, ...)`.
+* Fixed stack allocation after on-trace stack check.
+* Fixed recording of `__concat` metamethod that throws an error.
+* Fixed bit op coercion in `DUALNUM` builds.
+* Fixed 64-bit shift fold rules.
+* Fixed loop optimizations for cdata arguments of vararg FFI functions.
+
+### Lua
+
+* Fixed a memory leak in `xlog.pairs()` on failure to open the file (gh-10479).
+* Fixed a memory leak in `iconv.new()` (gh-10487).
+* Now `fun.chain` works correctly with iterators without `param`.
+* Now `fun.drop_while` supports stateful iterators.
+* Populated the `fun` module with the missing `maximum_by` alias
+  of `max_by`.
+* Now `fun.nth` and `fun.length` work correctly with Luafun iterators.
+
+#### Datetime
+
+* Fixed a bug that caused `datetime.parse()` ignore `tzoffset`
+  option if a custom format was used (gh-8333).
+* Forbid non-integers in `datetime.new()` (gh-10391).
+* Forbid non-integers in `:set()` and `datetime.parse()` (gh-10391).
+* Implemented the `tz` option in `datetime:parse()` (gh-10420).
+
+### SQL
+
+* Fixed a bug when an SQL count statement wasn't tracked by MVCC if it was
+  the first in a transaction (gh-10825).
+
+#### Datetime
+
+* Forbid non-integers in SQL's `CAST({}) AS datetime` (gh-10391).
```

Comment:
Nit: I don't think that a single item deserves its one fourth level section. I would include it to `SQL`.